mappings/net/minecraft/client/gui/screen/ingame/AbstractSignEditScreen.mapping

```diff
@@ -0,0 +1,28 @@
+CLASS net/minecraft/unmapped/C_dqqnwcwi net/minecraft/client/gui/screen/ingame/AbstractSignEditScreen
+	FIELD f_bhqmyyga lines [Ljava/lang/String;
+	FIELD f_lgnjllmz selectionManager Lnet/minecraft/unmapped/C_vyadpylu;
+	FIELD f_ojnprmbf ticks I
+	FIELD f_uinaqszz currentLine I
+	FIELD f_xlaqfmub signType Lnet/minecraft/unmapped/C_xlaykyai;
+	FIELD f_xpqrwatb signBlockEntity Lnet/minecraft/unmapped/C_axouvzlg;
```

Comment:
This could be shortened to just `blockEntity`

Reply:
```suggestion
	FIELD f_xpqrwatb sign Lnet/minecraft/unmapped/C_axouvzlg;
```